### PR TITLE
fix issue #267: col val outside range error

### PR DESCRIPTION
**Pull request by hiberabyss**

### CODE REVIEW

File: lua/hop/window.lua [hunk 1]
```diff
@@ -55,7 +55,7 @@ function M.get_window_context(multi_windows)
   local cur_hbuf = vim.api.nvim_win_get_buf(cur_hwin)
   all_ctxs[#all_ctxs + 1] = {
     hbuf = cur_hbuf,
-    contexts = { window_context(cur_hwin, {vim.fn.getcurpos()[2], vim.fn.getcurpos()[5]} ) },
+    contexts = { window_context(cur_hwin, vim.api.nvim_win_get_cursor(cur_hwin)) },
```
Comment (phaazon) [inline lua/hop/window.lua:58]: That was changed lately to support unicode. See #248. If you’ve found a regression introduced by this PR, the fix is probably something else.

**Review by phaazon — COMMENTED** [review]
As explained in my comment, you’re reverting a fix introduced in #248, so I can’t accept this PR.

Also, you didn’t link to any issue, and you didn’t explain what kind of problem you try to solve. I’m reading PRs on my spare-time and I don’t have the time to guess problems (even less to reproduce them). People send PRs and open issues all the time and it can be very time consuming to just try to review (read-only) the code of others, especially if people don’t provide any context or explanation.

Feel free to reopen with more context, or a linked issue. Also, I suggest you have a look at the [contributor guidelines of the project](https://github.com/phaazon/hop.nvim/blob/master/CONTRIBUTING.md#process).

**Comment by hiberabyss** [comment]
The PR linked to the issue #267 obviously.

**Comment by hiberabyss** [comment]
You could easily reproduce the problem via `HopLineAC` on empty line.

**Pull request closed without merging**